### ISSUE TITLE
HeroImage call to action secondary contrast

### ISSUE THREAD

**Issue by chrisrhymes** [FutureNorthants/northants-design-system#502]
When the HeroImage call to action is set to secondary and background box is set to true, the text contrast does not meet the AAA standard. Set the button background to white to meet the AAA contrast ratio. 

Box:
background-color: rgba(245, 245, 245, 0.95);

Button:
color: rgb(56, 88, 137);
background: transparent;

https://index.silktide.com/website/west-northamptonshire/july-2024/recommendations/aaa-text-contrast
